container_images/pytest/main.py

```diff
@@ -67,12 +91,19 @@ def setup_execution_root(package_root: str) -> str:
   Returns:
     Absolute path to execution root.
   """
-  assert os.path.exists(package_root) and os.path.isabs(package_root), \
-    f"Expected {package_root} to exist and to be an absolute path."
+  assert cfg.package_root.exists(), \
+      f"Expected {cfg.package_root} to exist and to be an absolute path."
```

Comment:
nit: why did this indentation change?

Reply:
poorly-managed autoformatter on my part. Reverted to the previous indent level.